app/controllers/services_controller.rb

```diff
@@ -13,7 +13,11 @@ def index
   # GET /services/1
   # GET /services/1.json
   def show
-    @related_services = Service.where(service_category: @service.service_category).where.not(id: @service.id)
```

Comment:
These changes seem unrelated to Finding Aid?

Reply:
Reverted this. I think these come from other work. Not sure why included here.

Reply:
Looks like the line was just moved.

Reply:
This and the one below for services show page are work that was aborted and it seems the branch was not reset properly before continuing work on unrelated issue of finding aids. Sloppy work. Sorry.

Reply:
Updated to current master version.